### ISSUE TITLE
Not working with Vercel CLI 23.0.1

### ISSUE THREAD

**Issue by styxlab** [Pytal/deploy-env#84]
Thanks a lot for this tool, I would like to use it with the latest vercel CLI. I am getting the following error:

```
Error! Environment Variable was not found.
Error! The Environment "API_URL" is invalid. It must be one of: <production | preview | development>.
```

Looking at the code, the issue come from a change in the `vercel cli`. Older CLI has:

```
 Vercel CLI 21.0.0
 vercel env [options] <command>
 add     [type] [name] [environment]     Add an Environment Variable (see examples below)
```

New CLI has:

```
Vercel CLI 23.0.1
vercel env [options] <command>
add     [name] [environment] [gitbranch]  Add an Environment Variable (see examples below)
```
which means `[type]`has been removed (`[gitbranch]`has been added).

Could you please update this project to the latest CLI version?

**Comment by Pytal** [comment]
Fixed in [v3.0.0](https://github.com/Pytal/deploy-env/releases/tag/v3.0.0)

**Comment by Pytal** [comment]
Fixed in [v3.0.0](https://github.com/Pytal/deploy-env/releases/tag/v3.0.0)
